runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityPlugin.java

```diff
@@ -421,4 +426,54 @@ private void onTileObject(Tile tile, TileObject oldObject, TileObject newObject)
 			}
 		}
 	}
+
+	@Subscribe
+	public void onMenuEntryAdded(MenuEntryAdded event)
+	{
+		if (!config.showShortcutLevel())
+		{
+			return;
+		}
+
+		if (event.getTarget() == null || event.getTarget() == "")
```

Comment:
This isn't necessary.

Reply:
Great suggestions, thanks for your input

Reply:
why did you mark this resolved